content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -142,20 +144,189 @@ id
 The output is similar to this:
 
 ```none
-uid=1000 gid=3000 groups=2000
+uid=1000 gid=3000 groups=2000,3000,4000
 ```
 
 From the output, you can see that `gid` is 3000 which is same as the `runAsGroup` field.
 If the `runAsGroup` was omitted, the `gid` would remain as 0 (root) and the process will
 be able to interact with files that are owned by the root(0) group and groups that have
-the required group permissions for the root (0) group.
+the required group permissions for the root (0) group. You can also see `groups`
+contains the group IDs which are specified by `fsGroup` and `supplementalGroups` other
+than `gid`.
 
 Exit your shell:
 
 ```shell
 exit
 ```
 
+### Implicit group memberships defined in `/etc/group` in the container image
+
+By default, kubernetes merges group information for the container's primary user defined in
```

Comment:
Maybe clearer (since "merges" suggests two things will be listed:

"""
By default, kubernetes merges group information from the Pod with information defined in `/etc/group` in the container image.
"""

Reply:
fixed in edd8ab30be